homeassistant/components/persistent_notification.py

```diff
@@ -92,12 +114,29 @@ def create_service(call):
 
         hass.states.async_set(entity_id, message, attr)
 
+    @callback
+    def dismiss_service(call):
+        notification_id = call.data.get(ATTR_NOTIFICATION_ID)
+
+        if notification_id is not None:
+            entity_id = ENTITY_ID_FORMAT.format(slugify(notification_id))
+        else:
+            entity_id = async_generate_entity_id(
```

Comment:
Actually, notification id is required in the schema so this branch will never be hit.